askama_derive/src/unescape.rs

```diff
@@ -0,0 +1,158 @@
+// This code is a simplified version of the `rustc_lexer` unescape.
```

Comment:
Hm, sorry, but I quite dislike this file. It kinda gives me paranoia that we miss some obscure corner case, which could lead to the rise of Skynet. :)

Reply:
Don't believe in me, believe in the rust compiler people who wrote it. :laughing:

Reply:
Yeah, but then by copying this we no longer benefit from their continued maintenance.

Reply:
Code hasn't changed in a long time, so we should be fine on that end. But yes, would be much better if we could get the "string value" of a raw string directly from proc-macro crate instead of having to handle it ourselves...